bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+stopped to make progress may be closed.
+
+When the BIP draft is complete, a BIP editor will assign a BIP type (see below), and merge it to the BIPs repository.
+After the BIP is merged to the repository, its main focus should no longer shift significantly, even while the champion
+may continue to update the draft as necessary in the git repository. Updates to drafts by the champion should also be
+submitted as pull requests.
+
+#### Nominated[^nominated]
+
+When the champion is confident that their BIP represents a net-improvement, is
+clear, comprehensive, has a working reference implementation (if applicable),
+and is ready for adoption by the Bitcoin community, they may update the BIP’s
+status to Nominated to indicate that they recommend adoption or implementation
+of the BIP. Where applicable, the champion must ensure that any proposed
+specification is solid, not unduly complicated, and definitive. Subsequently, a
+BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
+omissions in the specification, add test vectors for edge-cases, or address other issues discovered as the BIP is
+adopted.
+
+As time passes, substantial changes or changes that interfere with existing implementations are increasingly heavily
+discouraged.
+
+A Nominated BIP can only transition to Active or to Abandoned. If a Nominated BIP is found to need substantial
+functional changes, it should be moved to Abandoned and a new BIP should be drafted instead.
+
+#### Active
+
+At the earliest three months after a BIP has been moved to Nominated, a BIP may be advanced to Active upon request of
+any community member by providing evidence that the idea described in the BIP is in active use. Such evidence includes
+for example: two or more projects having deployed support for the BIP in mainnet software releases, a soft fork
+proposal’s activation criteria having been met on the network, or rough consensus for the BIP having been demonstrated.
+
+At that point, the BIP should be considered final and any breaking changes to the BIP should be proposed as a new
+separate BIP.
+
+##### Process BIPs
+
+A process BIP may change status from Nominated to Active when it achieves rough consensus on the mailing list. Such a
+proposal is said to have rough consensus if it has been open to discussion on the development mailing list for at least
+one month, and no person maintains any unaddressed substantiated objections to it. Addressed or obstructive objections
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. An Active Process BIP may be modified indefinitely as long as a proposed modification
+has rough consensus per the same criteria.
+
+##### Revisions
+
+Any changes to a BIP after it has transitioned to Nominated or Active are tracked with date and description in a
+Change Log section.
```

Comment:
The "Change Log" section should be added to the specification of the sections above.

Reply:
Added a description of the Change Log section.